services/ceo_dispatcher.py

```
@@ -267,6 +275,68 @@ async def delegate(
+        """Post-hoc independent re-check when a completed task touched a
+        risky module. Advisory only — attaches findings to *result* without
+        rolling back already-applied changes. Off by default.
+        """
+        if os.environ.get("AGENT_CROSS_VERIFY_ENABLED", "false").strip().lower() not in ("true", "1", "yes", "on"):
+            return
+        changed_files = _merge_changed_files(specialists_out)
+        if not changed_files:
+            return
+        try:
+            from agent.verification_strategies import cross_verify, touches_risky_module
+        except Exception:  # nosec B110 -- optional capability
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)